roles/kube_vip/defaults/main.yml

```diff
@@ -20,3 +20,18 @@ kube_vip_interface: "{{ keepalived_interface | default(kubernetes_keepalived_int
 
 # IP address to use for kube-vip
 kube_vip_address: "{{ keepalived_vip | default(kubernetes_keepalived_vip) }}"
+
+# use bgp for kube-vip
+kube_vip_bgp_enabled: false
+
+# BGP peers, if BGP is enabled
+kube_vip_bgp_peers: 192.168.0.10:65000::false,192.168.0.11:65000::false
+
+# BGP peer as, if BGP is enabled
+kube_vip_bgp_peeras: "65000"
+
+# BGP as, if BGP is enabled
+kube_vip_bgp_as: "65000"
+
+# BGP router id, if BGP is enabled
+kube_vip_bgp_routerid: 192.168.0.2
```

Comment:
Let's keep this commented, so that users will get an "undefined" error if they don't set it, making it a better experience when someone puts the wrong value.  You can keep it commented as an example.

Reply:
made change